HISTORY.md

```diff
@@ -1,4 +1,42 @@
-# Changelog
+## [1.1.0](https://github.com/theforeman/puppet-pulpcore/tree/1.1.0) (2020-06-30)
```

Comment:
I only add things to the changelog if otherwise the tool can't generate a correct changelog. Was this the case here? (noticed the same thing in puppet-pulp).

Reply:
I've been burned by weird stuff showing up in changelogs enough times that I just made updating HISTORY.md a part of my process. I think it makes sense to keep it updated to the (n-1) release anyway since it just feels weird if we are on version 9.0.0 of a module but HISTORY.md only contains changes through 5.0.0

Reply:
I don't mind it, but I'm wondering if it makes sense to go all the way and copy the newest one as well (1.2.0 in this case). Thoughts?